### ISSUE TITLE
[Slider] voice over should announce value after adjusting the slider

### ISSUE THREAD

**Issue by jmdetloff** [material-components/material-components-ios#4289]
Voice over currently announces the new value after a double tap + swipe up or down to adjust, but not after a double tap + hold + pan to adjust. This is important because for discrete sliders that second method is currently the only method to adjust the slider value.
